src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
```

Comment:
* Layouts begin with an underscore, so `._card`
* This, again, needs a more descriptive component name, and the component name and the file name need to match